src/periphery/Timelock.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/governance/TimelockController.sol";
+
+contract Timelock is TimelockController {
```

Comment:
This contract doesn't need to exist, TimelockController isn't abstract and can be deployed directly